### ISSUE TITLE
Standalone version

### ISSUE THREAD

**Issue by iManojrk** [antonmedv/numbr#6]
I would love to use a standalone version of the app without have to use as a website or as an extension.

**Comment by antonmedv** [comment]
Yep. This is on my  todo list. Right now there is a Chrome extension.

**Comment by ivanalejandro0** [comment]
Maybe what could be done is to turn the app into a PWA.
As a full PWA, Numbr could be installed and used offline.
You would still be using a website, but after installing it (on desktop and mobile) you can use it kind of like an app and it won't need an internet connection.
This may not be a trivial amount of work, but most likely would be much easier than making Numbr standalone by other means like shipping it as a desktop/mobile app.
A PWA has its drawbacks and limitations, though... but it may be worth considering along other alternatives if making it standalone is a goal for the project.

**Comment by Kaylebor** [comment]
PWAs have been removed from desktop Firefox (which is stupid), so that will only ever work with Chrome.
I'm not explicitly asking for Firefox support, even though that'd be cool, but it's something to keep in mind I believe.

**Comment by antonmedv** [comment]
Now Safari and Chrome both supports PWA, and both works great. On macOS and Windows. This should cover 99% of users.

**Comment by antonmedv** [comment]
<img width="1141" alt="Screenshot 2023-11-21 at 16 33 33" src="https://github.com/antonmedv/numbr/assets/141232/d17fda90-ebd9-4e8e-820d-b1d36abb330a">

Firefox usage is less than half of a precent.

**Comment by Kaylebor** [comment]
Fair enough, I know that adding one more platform to maintain isn't as easy as just clicking a button, even if Firefox extensions are supposedly in a compatible format (I haven't developed extensions, so I don't want to say much more on this; it's likely not as easy as just redeploying it).

It'd be interesting to know if Firefox users like myself aren't interested in the tool, or if we just bounce off because of lack of Firefox support... I don't know how to measure that though, but I know that I haven't even tried the app, precisely because the home page screams "Chrome-only"; then again, I think global Firefox usage is sub-5% nowadays, so it won't ever be a big percentage of users.
Plus, I suspect there's an overlap between Firefox users and users who'd prefer a separate app 👀

I'll just wait for a standalone version of the app then. One browser is already enough of a resource hog on my system, and Firefox is far more customizable (I specially like [Sidebery](https://addons.mozilla.org/en-US/firefox/addon/sidebery/))

**Comment by antonmedv** [comment]
I will think about adding a standalone version of the app in the future. Maybe via tauri. 

Another thing about Firefox: it gas bags, not stuff what works differently from chrome, just really annoying bugs, bugs 5-7 years old bugs in big tracker. And dev tools in chrome definitely two heads ahead of firefox.

**Comment by dsernst** [comment]
In MacOS, anyone can now [create a standalone app](https://support.apple.com/en-us/104996) by opening the site in Safari, then going to `File > Add to Dock`:

<img width="402" alt="image" src="https://github.com/antonmedv/numbr/assets/6340841/5dfa5c55-5e7e-49f9-b695-23486cfad23b">

**Comment by gustavotrott** [comment]
You can build a standalone offline version using this fork:
https://github.com/gustavotrott/numbr

```
npm install
npm run build:static
```

All you need is at `/dist` directory.

Demo: https://gustavotrott.github.io/numbr/dist/
